lesson01.trivia/index.html

```diff
@@ -1,22 +1,73 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang='en'>
+
 <head>
     <title>Simple Trivia</title>
-    <meta charset="utf-8">
-    <meta name="viewport"
-          content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta charset='utf-8'>
+    <meta name='viewport' content='width=device-width, initial-scale=1, shrink-to-fit=no'>
     <style>
         body {
+            background-image: url('https://mk0astronomynow9oh6g.kinstacdn.com/wp-content/uploads/2015/11/ejected_planet_940x500.jpg');
+            background-size: cover;
+            background-color: darkslateblue;
+            color: white;
+            font-family: Arial, Helvetica, sans-serif;
+        }
 
+        .question li,
+        .question .result,
+        .question .solution {
+            font-size: 1.5em;
+            line-height: 1.5em;
+        }
+
+        .question li {
+            line-height: 1.5em;
```

Comment:
can you explain usage of `em` units?

Reply:
As much as I know, em means relative to the font size of the element, which is 1em by default.